File: readBooks.js
const fs = require('fs');

fs.readFile('books.json', 'utf8', (err, data) => {
    if (err) {
        console.error(err);
        return;
    }

    const bookData = JSON.parse(data);

    // Access all books
    bookData.Books.forEach(book => {
        console.log(`
Title: ${book.title}
Author: ${book.author}
Year: ${book.publicationYear}
Genres: ${book.genres.join(", ")}
        `);
    });
});

File: readBooks.py
import json

with open('books.json') as f:
    data = json.load(f)

for book in data['Books']:
    print(f"""
Title: {book['title']}
Author: {book['author']}
Year: {book['publicationYear']}
Genres: {', '.join(book['genres'])}
    """)